spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -1,2 +1,217 @@
-[[repository]]
-== Repository
\ No newline at end of file
+
+= Repository
+
+In DDD, the repository pattern encapsulates the logic required to access data sources. They consolidate data access functionality, providing better maintainability and decoupling the infrastructure or technology used to access databases from the domain model layer.
```

Comment:
The "DDD" acronym should be spelled out as "Domain-driven design".

In "They consolidate", the word "They" will be understood to refer to "data sources" as written.  This should be clarified as, "The repository pattern consolidates".